ccontainermain.go

```diff
@@ -621,7 +623,7 @@ func main() {
 	if exeStop != "" {
 		_, err := stopExtraService(exeStop)
 		if err != nil {
-			log.Printf("\nError shutting down eXtra service: %s; err: \n", exeStop, err)
+			log.Printf("\nError shutting down eXtra service: %s; err: %s\n", exeStop, err)
```

Comment:
It's just looks like a typo, because in string we see only one place for variable, but given two.
and even golint, show some warning about this line, and some others